tests/test-listener-manager.c

```diff
@@ -9,7 +9,9 @@
 
 
 #include <arpa/inet.h>
+#include <netinet/in.h>
 
+#include <ell/util.h>
```

Comment:
This breaks with a recent ELL - needs the wrapper pragmas for these ELL includes

Reply:
Fixed in commit e80db71.